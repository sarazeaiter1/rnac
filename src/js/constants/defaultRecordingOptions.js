const defaultRecordingOptions = {
  SampleRate: 44100.0,
  Channels: 2,
  AudioQuality: 'High',
  AudioEncoding: 'ima4',
  OutputFormat: 'mpeg_4',
  MeteringEnabled: false,
  AudioEncodingBitRate: 32000
};

export default defaultRecordingOptions;
