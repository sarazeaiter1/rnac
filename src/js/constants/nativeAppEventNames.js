const NativeAppEventEmitter = {
  recordingProgress: 'recordingProgress',
  recordingFinished: 'recordingFinished'
};

export default NativeAppEventEmitter;
