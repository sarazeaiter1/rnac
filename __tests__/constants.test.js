import defaultRecordingOptions from '../src/js/constants/defaultRecordingOptions';
import nativeAppEventNames from '../src/js/constants/nativeAppEventNames';

describe('constants', () => {
  test('defaultRecordingOptions', () => {
    expect(defaultRecordingOptions).toMatchSnapshot();
  });
  test('nativeAppEventNames', () => {
    expect(nativeAppEventNames).toMatchSnapshot();
  });
});
