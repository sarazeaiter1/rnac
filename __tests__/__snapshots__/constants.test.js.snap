// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants defaultRecordingOptions 1`] = `
Object {
  "AudioEncoding": "ima4",
  "AudioEncodingBitRate": 32000,
  "AudioQuality": "High",
  "Channels": 2,
  "MeteringEnabled": false,
  "OutputFormat": "mpeg_4",
  "SampleRate": 44100,
}
`;

exports[`constants nativeAppEventNames 1`] = `
Object {
  "recordingFinished": "recordingFinished",
  "recordingProgress": "recordingProgress",
}
`;
